entries/s/schibsted.com.json

```
@@ -24,12 +24,15 @@
       "godt.no",
       "minmote.no",
       "pent.no",
-      "tek.no"
+      "tek.no",
+      "oikotie.fi",
+      "tori.fi"
     ],
     "tfa": [
       "sms",
       "totp"
     ],
+    "documentation": "https://login.schibsted.com/profile-pages/faq",
```

The entire page is rather long and it is primarily this part that's of interest.
```suggestion
    "documentation": "https://login.schibsted.com/profile-pages/faq#how-to-activate-and-deactivate-2fa",
```